functional_tests/scylla_operator/test_functional.py

```diff
@@ -87,6 +93,128 @@ def test_single_operator_image_tag_is_everywhere(db_cluster):
         f"Pods: {yaml.safe_dump(pods_with_wrong_image_tags, indent=2)}")
 
 
+@pytest.mark.required_operator("v1.11.0")
+def test_deploy_quasi_multidc_db_cluster(db_cluster: ScyllaPodCluster):  # pylint: disable=too-many-locals,too-many-statements,too-many-branches
+    """
+    Deploy 2 ScyllaCluster objects and combine them into a single DB cluster.
```

Comment:
IIUC would be worth to add info that quasi-multidc means one cluster but two namespaces (and using PodIPs for connectivity as you already mentioned)

Also add info what is tested here - verify peers for each node lists nodes from both dc's.